main.py

```diff
@@ -16,11 +19,15 @@ def __init__(self, checkpoint, original_dataset, encoding_scheme: str):
         checkpoint : (int) how often to store the reference frame
         original_dataset: (OriginalDataset) the original dataset
         """
-        super().__init__()
+        super().__init__(self)
         self.checkpoint = checkpoint
         self.original_dataset = original_dataset
+
+        self.enlarge_factor = 35
```

Comment:
you can add this parameter to the function call, so that we dont have to come and modify here for changes